examples/qr-generator-demo/qr-generator-demo.py

```diff
@@ -1,9 +1,13 @@
 import pandas as pd
 import qrcode
-
+from PIL import Image
+from PIL import ImageFont
+from PIL import ImageDraw
 
 def image_generator(csv_file_path):
     df = pd.read_csv(csv_file_path, usecols=['employee_name', 'equipment_name', 'equipment_id'])
+    x=140
+    y=10
```

Comment:
top_left_text_position = (140, 10)
move this definition closer to its usage, i.e. line 22

Reply:
@NikitaKit1998  ^^